met-web/src/services/commentService/index.tsx

```diff
@@ -40,6 +40,22 @@ export const getCommentsPage = async ({
     );
 };
 
+export const getSubmissionComments = async (submissionId: number): Promise<Comment[]> => {
+    const url = replaceUrl(Endpoints.Comment.GET_SUBMISSION, 'submission_id', String(submissionId));
+    if (!submissionId || isNaN(Number(submissionId))) {
+        return Promise.reject('Invalid Submission Id ' + submissionId);
+    }
```

Comment:
remove this plz if you think there is no additional value , i think it gets carried over from the previous code...

Reply:
removed